### DISCUSSION TITLE
Postgresql 15 support?

### DISCUSSION THREAD

**Discussion by cjpeterein** [sqlalchemy/sqlalchemy#10235] [Usage Questions] [answered]
Does sqlalchemy support postgresql 15?  Related issue for postgresql 14: https://github.com/sqlalchemy/sqlalchemy/issues/7647 , which said the docs just needed to be updated.

I see psycopg supports postgresql 15: https://www.psycopg.org/docs/install.html

https://docs.sqlalchemy.org/en/20/dialects/postgresql.html says the highest tested in CI is 14.  I tried looking at the CI code, but I wasn't able to figure out how it tests multiple postgresql versions.

**Comment by zzzeek** [comment, marked as answer]
sure, we support PG 15 and pg15 is in our current CI.  the config comes from here: https://github.com/sqlalchemyorg/ci_containers/blob/main/config.yml#L79 

so the doc is out of date and im not sure why we even talk about the upper PG version as they release new major versions quite frequently

**Reply by CaselIT** [reply]
Yes we should probably mention that that versuon is not a upper bound for support

**Reply by zzzeek** [reply]
the docs for all backends will be updated on the site in the next hour, see fc33a6b159f88a7374d

**Comment by CaselIT** [comment]
Hi,

Same dial as the referenced issue. Pg15 is already supported and we also added support for some of the new features, like the nulls not distinct on indexes and unique constraints.

A pr to update the docs would we appreciated!
